pkg/chunked/cache_linux.go

```diff
@@ -154,6 +162,19 @@ fallback:
 	return buf, nil, err
 }
 
+func getBinaryDigest(stringDigest string) ([]byte, error) {
+	d, err := digest.Parse(stringDigest)
+	if err != nil {
+		return nil, err
+	}
+	digestBytes, err := hex.DecodeString(d.Hex())
+	if err != nil {
+		return nil, err
+	}
+	digest := append([]byte(d.Algorithm()+":"), digestBytes...)
```

Comment:
with that I get:

```
./cache_linux.go:174:47: too many arguments in call to append
	have ([]byte, number, []byte)
	want ([]byte, ...byte)
```

Reply:
I’m sorry, I have misremembered the language spec. So this would have to be two separate `append` calls.

Anyway, I think this micro-optimization can wait for the updated shape, if any, of `generateTag`/`getBinaryDigest`, per the other comments.